### ISSUE TITLE
nvda.press('ArrowRight') doesn't work

### ISSUE THREAD

**Issue by AnyMa** [guidepup/guidepup#63]
Method nvda.press('ArrowRight') doesn't work. It just does nothing. Normal NVDA works fine for that function.

**Comment by AnyMa** [comment]
The fix worked well! Thanks Craig!
